pymysql/connections.py

```diff
@@ -1191,4 +1227,70 @@ def _get_descriptions(self):
         assert eof_packet.is_eof_packet(), 'Protocol error, expecting EOF'
         self.description = tuple(description)
 
+
+class LoadLocalFile(object):
+    def __init__(self, filename, connection):
+        self.filename = filename
+        self.connection = connection
+
+    def _print_warnings(self):
+        self.connection._execute_command(COMMAND.COM_QUERY, 'SHOW WARNINGS')
+        warnings = MySQLResult(self.connection)
+        warnings.read()
```

Comment:
If we remove this then we will not have the same functionality as mysqldb, which I thought was a goal of pymysql, no? The user will have no idea if there were any warnings returned after their file was loaded, which seems a bit dangerous to me. 

Also, out of curiosity, how do you expect print() to fail?